### PR TITLE
Check account type

### PR DESCRIPTION
**Pull request by rao-kartik**

### CODE REVIEW

File: Week-3/2_CheckAccountType/Readme.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Write a small solidity function that - 
+
+1. Accepts 1 address as argument 
+2. Returns boolean False if that address is an EOA 
+3. Returns TRUE if that address is a Contract
+
+## Brief
```
Comment (mahima-manik) [inline Week-3/2_CheckAccountType/Readme.md:7]: You can add a line about difference between EOA and contract address

File: Week-3/2_CheckAccountType/Readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Solved using two ways:
+
+1. Using open zepplin library
```
Comment (mahima-manik) [inline Week-3/2_CheckAccountType/Readme.md:10]: Explain what extcodesize is doing?

**Review by mahima-manik — CHANGES_REQUESTED** [review]
Calculator contract is repeated on this branch

File: Week-3/2_CheckAccountType/CheckAccountType.sol [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+//         if(_addr.isContract()){
+//             return "Contract Account";
+//         }
+//         return "EOA";
+//     }
+// }
+
+contract CheckAccount {
+
+    function checkAccountType (address _addr) external view returns(string memory) {
```
Comment (SrikanthAlva) [inline Week-3/2_CheckAccountType/CheckAccountType.sol]: Note: This is one of the primary ways to check for Contract Account and EOA. However, this check can be bypassed if the checkAccountType function is called inside the constructor of another contract.

File: Week-3/2_CheckAccountType/Readme.md (hunk 1 above)
Comment (rao-kartik) [reply Week-3/2_CheckAccountType/Readme.md:7]: Done

File: Week-3/2_CheckAccountType/Readme.md (hunk 2 above)
Comment (rao-kartik) [reply Week-3/2_CheckAccountType/Readme.md:10]: Done

**Review by SrikanthAlva — APPROVED** [review]

**Pull request merged**